apps/glusterfs/dbcommon_test.go

```diff
@@ -0,0 +1,216 @@
+//
```

Comment:
Why dbcommon_test.go ? Why not something like delete_bricks_empty_path_test.go ?

Reply:
it is recommended that the test file has same name as the corresponding file containing function. I will move functions from app_db.go and dbcommon.go which belong to db tooling to another file in next PR.